common/client-core/src/client/base_client/mod.rs

```diff
@@ -191,6 +195,15 @@ where
         self
     }
 
+    pub fn with_stored_topology<P: AsRef<Path>>(
```

Comment:
No `#[must_use]` ?

Reply:
no, because it's already implicitly a `#[must_use]` due to the `Result` having it (and clippy actually complains if you add it there)